src/main/java/com/wavefront/sdk/common/Constants.java

```diff
@@ -79,4 +79,9 @@ private Constants() {
    * Tag key for defining a component.
    */
   public static final String COMPONENT_TAG_KEY = "component";
+
+  /**
+   * Name prefix for internal diagnostic metrics for Wavefront SDKs.
+   */
+  public static final String SDK_METRIC_PREFIX = "~sdk.java";
```

Comment:
this can be ~sdk.java.core.sender

Reply:
I was thinking that other SDKs can use this as their base prefix too.